doc/nrf/device_guides/working_with_nrf/nrf52/developing.rst

```diff
@@ -215,6 +223,7 @@ When performing a FOTA update when working with the Bluetooth Mesh protocol, use
 
 * DFU over Bluetooth Mesh using the Zephyr Bluetooth Mesh DFU subsystem.
 * Point-to-point DFU over Bluetooth Low Energy as described in `FOTA over Bluetooth Low Energy`_ above.
+  The Bluetooth Mesh samples use random HCI identities. See the related notes in the Testing steps section.
```

Comment:
```suggestion
  The Bluetooth Mesh samples use random HCI identities.
  See the related notes in the Testing steps section.
```

Reply:
Actually, it wouldn't hurt to reference to it, like we do in the bullet point above. I.e. `Testing steps`_ section.